diff --git a/meson.build b/meson.build
index a56bb1f37c..7cd3b195f9 100644
--- a/meson.build
+++ b/meson.build
@@ -26,6 +26,9 @@ libplacebo = dependency('libplacebo', version: '>=6.338.2',
                default_options: ['default_library=static', 'demos=false'])

+# fftools-ffi
+libfftools_ffi = dependency('fftools-ffi')
+
 libass = dependency('libass', version: '>= 0.12.2')
 
 # the dependency order of libass -> ffmpeg is necessary due to
@@ -33,11 +36,12 @@ dependencies = [libass,
                 libavcodec,
                 libavfilter,
                 libavformat,
                 libavutil,
                 libplacebo,
                 libswresample,
-                libswscale]
+                libswscale,
+                libfftools_ffi]
 
 # Keeps track of all enabled/disabled features
 features = {
@@ -54,3 +58,6 @@ # generic sources
 sources = files(
+    ## fftools-ffi hack
+    'fftools-ffi.c',
+
     ## Audio
 
 
diff --git a/fftools-ffi.c b/fftools-ffi.c
new file mode 100644
index 0000000..6cd4258
--- /dev/null
+++ b/fftools-ffi.c
@@ -0,0 +1,6 @@
+#include "fftools-ffi/dart_api.h"
+
+void* a = FFToolsFFIInitialize;
+void* b = FFToolsFFIExecuteFFmpeg;
+void* c = FFToolsFFIExecuteFFprobe;
+void* d = FFToolsCancel;
